.github/workflows/integration.yml

```diff
@@ -58,17 +57,34 @@ jobs:
         path: .venv
         key: python-${{ matrix.python-version }}-pydeps-${{ hashFiles('**/poetry.lock') }}
 
-    # Begin the integration test
-    # Commenting this out for now while we improve it.
-    # - name: Poetry Install
-    #   run: poetry install --no-interaction --no-root
-    #   if: steps.cache-deps.outputs.cache-hit != 'true'
-    # - name: Install Requirements
-    #   working-directory: ./pcweb
-    #   run: poetry run pip install -r requirements.txt
-    # - name: Init Website
-    #   working-directory: ./pcweb
-    #   run: poetry run pc init
-    # - name: Run Website
-    #   working-directory: ./pcweb
-    #   run: timeout 1m poetry run pc run || exit 0
\ No newline at end of file
+    - name: Poetry Install
+      run: poetry install --no-interaction --no-root
+      if: steps.cache-deps.outputs.cache-hit != 'true'
+    - name: Install Requirements
+      working-directory: ./pynecone-examples/counter
+      run: poetry run pip install -r requirements.txt
+    - name: Init Website
+      working-directory: ./pynecone-examples/counter
+      run: poetry run pc init
+    - name: Run Website and check for errors
+      working-directory: ./pynecone-examples/counter
+      run: |
+        curl -fsSL https://bun.sh/install | bash -s -- bun-v0.5.5
+        timeout 2m poetry run pc run &
```

Comment:
When we add `pcweb` let's run `pc run --env prod` since it will force compile all the pages to JS. For this app it doesn't matter since it's only a single app.